addons/website_forum/static/src/js/website_forum.js

```diff
@@ -448,6 +449,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         queueCountBadge.innerText = nbLeftInQueue;
         if (!nbLeftInQueue) {
             document.querySelector('.o_caught_up_alert').classList.remove('d-none');
+            document.querySelector('.o_wforum_btn_filter_tool').classList.add('d-none');
```

Comment:
This will cause a crash if a user doesn't upgrade the module to get the new xml as the class is not present yet. Could you therefore simply add this?

```suggestion
            document.querySelector('.o_wforum_btn_filter_tool')?.classList.add('d-none');
```